src/index.ts

```diff
@@ -20,7 +24,13 @@ import "./blocks/operators";
 import "./blocks/procedures";
 import "./blocks/sensing";
 import "./blocks/sound";
-import * as scratchBlocksUtils from "./scratch_blocks_utils";
+/* BEGIN RG ADDITION */
```

Comment:
This is necessary because the `scratch-gui` code wants to overwrite the `compareStrings` method:

https://github.com/scratchfoundation/scratch-gui/blob/develop/src/lib/blocks.js#L344